src/Robo/Commands/Vm/VmCommand.php

```diff
@@ -118,6 +124,31 @@ protected function install() {
    * @command vm:config
    */
   public function config() {
+    $drupalvm_install_dir = $this->askDefault("Where would you like to install Drupal-VM files?", $this->vmConfigDir);
```

Comment:
Hmm... That's fair, I'll put up another PR with an alternate treatment. Let me know what you like better...?
The only reason I would disagree with the prompt point is that the VM command already prompts the user to boot the VM. It's also a specific command that is run by a single user after the BLT project has already been set up. It's not like this would ever be run non-interactively, right?